galeracluster/source/sslconfig.rst

```diff
@@ -12,7 +12,9 @@ When you finish generating the SSL certificates for your cluster, you need to en
 Enabling SSL
 -----------------------
 
-There are three vectors that you can secure with :abbr:`SSL (Secure Socket Layer)`: the database server and client, replication traffic from Galera Cluster, and state snapshot transfers through ``stunnel``.
+There are three vectors that you can secure with :abbr:`SSL (Secure Socket Layer)`: the database server and client as well as replication traffic from Galera Cluster.
```

Comment:
There are three vectors that you can secure with SSL: The traffic between the database server and the client, the replication traffic within Galera Cluster and the State Snapshot Transfers (SST).